test/scrabble_test.rb

```diff
@@ -6,8 +6,8 @@
 
 class ScrabbleTest < Minitest::Test
   def test_it_can_score_a_single_letter
-    skip
     assert_equal 1, Scrabble.new.score("a")
     assert_equal 4, Scrabble.new.score("f")
   end
```

Comment:
Would like to see more testing here. I know that it seems like this takes away from your time to complete the task, but I actually think it helps you concentrate on one task at a time and will then help you move through those tasks more quickly.